### PR TITLE
fix upload file delay

### PR DESCRIPTION
**Pull request by aliabid94**
There was a delay between an uploaded file and then trigger for `change` and `upload` listeners. This was caused by the fact that files were actually getting uploaded *twice*, once in the svelte File component, and once in the client library when submitting data. This us because the client looks to see if there's any blob in the component data and uploads it. I removed the blob in the component so that the client does not resubmit - ideally we could let the client handle all uploading but the upload animation and dispatching of "change" and "submit" within the component won't work with that, so it's handled only in the component now. 

Fixes: #4385 

To test, run any demo that has a change or upload trigger on file, example below. Previously, you would see a lag between the finishing of the upload animation and triggering of the backend. You would also two network requests made "/upload" made sequentially. These should be fixed.

```python
import os
from zipfile import ZipFile
import gradio as gr


def zip_files(files):
    with ZipFile("tmp.zip", "w") as zipObj:
        for idx, file in enumerate(files):
            zipObj.write(file.name, file.name.split("/")[-1])
    return "tmp.zip"

with gr.Blocks() as demo:
    file = gr.File(label="a", file_count="multiple")
    zipped = gr.File(label="b")
    file.change(zip_files, file, zipped)

demo.queue().launch()
```

### CODE REVIEW

**Comment by pngwn** [comment]
Ah, my bad. Nice catch.

File: js/app/src/components/File/File.svelte [hunk 1]
```diff
@@ -80,10 +81,11 @@
 									file_data.orig_name = file_data.name;
 									file_data.name = response.files[i];
 									file_data.is_file = true;
+									file_data.blob = null;
```
Comment (pngwn) [inline js/app/src/components/File/File.svelte]: Think we just need to use `undefined` for all of these empty blob values and those errors.

Comment (pngwn) [reply js/app/src/components/File/File.svelte]: beat me to it.

**Review by pngwn — APPROVED** [review]
Tested and working. Looks good to me! Thanks @aliabid94!

**Comment by abidlabs** [comment]
Thanks guys! Will go ahead and merge in so that we can work on reducing the number of open PRs!

**Pull request merged**